README.md

```diff
@@ -18,33 +18,37 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-aws) to see addit
         <img src="https://img.shields.io/badge/discourse-browse_forum-red.svg?color=0052FF&labelColor=090422&logo=discourse" /></a>
 </p>
 
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-aws) to see additional examples and the API reference.
+
 ## Welcome!
 
 `prefect-aws` is a collection of pre-built Prefect tasks that can be used to quickly construct Prefect flows that interact with Amazon Web Services.
 
 ## Getting Started
 
-### Python setup
-
-Requires an installation of Python 3.7+
-
-We recommend using a Python virtual environment manager such as pipenv, conda or virtualenv.
-
-These tasks are designed to work with Prefect 2.0. For more information about how to use Prefect, please refer to the [Prefect documentation](https://orion-docs.prefect.io/).
+### Saving credentials to a block
 
-### Installation
+You will need to obtain AWS credentials in order to use `prefect-aws`.
 
-Install `prefect-aws`
+1. Refer to the [AWS Configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds) on how to retrieve your access key ID and secret access key
+2. Copy the access key ID and secret access key
+3. Create a short script, replacing the placeholders (or do so in the UI)
 
-```bash
-pip install prefect-aws
+```python
+from prefect_aws import AwsCredentials
+AwsCredentials(
+    aws_access_key_id="PLACEHOLDER",
+    aws_secret_access_key="PLACEHOLDER",
+    aws_session_token=None,  # replace this with token if necessary
+).save("BLOCK_NAME_PLACEHOLDER")
 ```
 
-A list of available blocks in `prefect-aws` and their setup instructions can be found [here](https://PrefectHQ.github.io/prefect-aws/#blocks-catalog).
-
-### AWS Authentication
-
-You will need to obtain AWS credentials in order to use these tasks. Refer to the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for authentication methods available.
+Congrats! You can now easily load the saved block, which holds your credentials:
+ 
+```python
+from prefect_aws import Block
+Block.load("BLOCK_NAME_PLACEHOLDER")
```

Comment:
```suggestion
from prefect_aws import AwsCredentials
AwsCredentials.load("BLOCK_NAME_PLACEHOLDER")
```

Reply:
Nice catch! Fixed the import too in the latest commit.